framework/platform/bpl/uci/arp/monitor/arp_monitor.h

```diff
@@ -88,7 +88,7 @@ class arp_monitor {
     std::list<std::shared_ptr<SProbeEntry>> m_lstProbe;
 
     // Buffer for reading ARP packets
-    char m_arrArpPacket[ETH_FRAME_LEN];
+    char m_arrArpPacket[ETH_FRAME_LEN]{0};
```

Comment:
I didn't know this syntax, I thought it had to be 

```suggestion
    char m_arrArpPacket[ETH_FRAME_LEN] = {0};
```

#til